source/val/Function.cpp

```diff
@@ -231,6 +245,11 @@ vector<BasicBlock*>& Function::get_blocks() { return ordered_blocks_; }
 const BasicBlock* Function::get_current_block() const { return current_block_; }
 BasicBlock* Function::get_current_block() { return current_block_; }
 
+BasicBlock* Function::get_pseudo_entry_block() { return &pseudo_entry_block_; }
```

Comment:
According to the style guide, simple getter methods should be just named as the corresponding field with trailing `_` stripped. But I see that all the getter functions are prefixed with `get` here. So, maybe for a later CL to convert them all.